bpo-44895: Introduce PYTHONDUMPREFSFILE variable for refcount dumping (#27767)
python/cpython: Doc/using/cmdline.rst

   shutting down the interpreter.

   Need Python configured with the :option:`--with-trace-refs` build option.

> .. envvar:: PYTHONDUMPFILE

Reviewer: "Dump file" is a generic name. What do you think of "Dump refs file" (PYTHONDUMPREFSFILE)? (and rename the PyConfig member to dump_refs_file).

   If set, Python will dump objects and reference counts still alive

>    after shutting down the interpreter into a file called `$FILENAME`.

Reviewer: ```suggestion
   after shutting down the interpreter into a file called *FILENAME*.
```

   shutting down the interpreter.

   Need Python configured with the :option:`--with-trace-refs` build option.

.. envvar:: PYTHONDUMPREFSFILE

>    If set, Python will dump objects and reference counts still alive
>    as file named `$PYTHONDUMPREFSFILE` still alive after shutting down
>    the interpreter.

Reviewer: ```suggestion
   If set, Python will dump objects and reference counts still alive
   after shutting down the interpreter into a file called `$PYTHONDUMPREFSFILE`.
```

Is it possible to use ``.. envvar:: PYTHONDUMPREFSFILE=FILENAME`` syntax and then use ``FILENAME`` in the text?
